runtime/compiler/optimizer/J9Inliner.cpp

```diff
@@ -855,7 +855,7 @@ bool TR_J9InterfaceCallSite::findCallSiteTargetImpl(
          kind = TR_InterfaceGuard;
          testType = TR_MethodTest;
          }
-      else
+      else if (!comp()->compileRelocatableCode() || comp()->getOption(TR_UseSymbolValidationManager))
```

Comment:
I don't think we need to skip this entire section for non-SVM AOT. It should suffice to set `useVftTestHeuristics` to false